logs/handler_test.go

```diff
@@ -103,12 +103,6 @@ func Test_GETRequestParsing(t *testing.T) {
 			err:             "",
 			expectedRequest: Request{Name: "foobar"},
 		},
-		{
```

Comment:
This test was a copy/paste duplicate of the one above it.  I probably accidentally included it during the initial implementation